### DISCUSSION TITLE
WARNING - Looking up qualities on plot takes more than 5 sec

### DISCUSSION THREAD

**Discussion by galgool** [Chia-Network/chia-blockchain#4484] [Support]
I’m getting many warnings of plot look up qualities times higher than 5 secs.
This is new. I haven’t got that in the past and I’m not getting any rewards for over a week.
I have 330 TiB and everything seems to work fine, so expectation to win one block every 2 days now, and a week ago one block every day.

Could the reason be the look up time? And Is there any way to fix this?

**Comment by lvcivs** [comment]
I believe this warning was added to the chia software very recently, so it does not necessarily mean that anything has changed on your end.

You can change the log level to INFO and then check the log entries, that way you will see precisely how long your lookups take and you will know whether it's an issue.

**Reply by galgool** [reply]
Harvester warnings:

2021-05-11T12:59:42.681 harvester chia.harvester.harvester: WARNING Looking up qualities on N:\final\plot-k32-2021-05-04-08-00-e577a48...................a485.plot took: 10.492086410522461. This should be below 5 seconds to minimize risk of losing rewards.

It says times should be under 5 sec

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
